core/modules/block/tests/block.translation.test

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/**
+ * @file
+ * Tests for block.module.
+ */
+
+class BlockTranslationTestCase extends BackdropWebTestCase {
+  protected $admin_user;
```

Comment:
Ideally new tests added should use `protected $profile = 'testing';` to cut down on the installation time. But if that is problematic we can keep using the standard profile.